src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchBar {
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  font-weight: 300;
+  color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchBar.big {
+  background-size: 19px;
+  background-position-x: 26px;
+  font-size: 16px;
+  text-indent: 62px;
+}
+
+.searchBar.small {
+  background-size: 11px;
+  background-position-x: 13px;
+  font-size: 14px;
+  text-indent: 33px;
+}
```

Comment:
do not increase specificity. if you want to have search bar, but you have two sizes, create a class .searchBar and two modifiers: searchBar--big and searchBar--small. put common styles in the searchBar and what is different in the modifiers.